### PR TITLE
added latex layer

### PR DESCRIPTION
**Pull request by spencerlyon2**
I'm very new to clojure (only used to contribute here) so I'm open to suggestions on the best way to add to the packages list based on a config value. I copied the pattern used in core for installing the nuclide packages.

### CODE REVIEW

File: src/cljs/proton/layers/lang/latex/core.cljs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+(def packages
+  (atom
+    [:latexer
+     :language-latex
+     :autocomplete-bibtex
+     :pdf-view]))
+
+(defmethod init-layer! :lang/latex [_ config]
+  (let [config-map (into (hash-map) config)]
+    (if (config-map "proton.lang.latex.use-latex-plus")
```
Comment (dvcrn) [inline src/cljs/proton/layers/lang/latex/core.cljs]: > proton.lang.latex.use-latex-plus

yeah we should definitely try to get the new config system (https://github.com/dvcrn/proton/issues/22) out anytime soon. I think all these config strings might get confusing 

Looking good to me

**Comment by dvcrn** [comment]
Hmm looks like a previous merge messed this up a little. Can you rebase `master`?

**Comment by spencerlyon2** [comment]
Rebased.

And yeah, I agree that the new config system will make these options much easier to set!

**Comment by dvcrn** [comment]
Merged :+1:

**Pull request merged**